src/Microsoft.TestPlatform.PlatformAbstractions/netcore/System/ProcessHelper.cs

```diff
@@ -42,6 +42,7 @@ public PlatformArchitecture GetCurrentProcessArchitecture()
             // case System.Runtime.InteropServices.Architecture.S390x:
             (Architecture)5 => PlatformArchitecture.S390x,
             (Architecture)8 => PlatformArchitecture.Ppc64le,
+            (Architecture)9 => PlatformArchitecture.RiscV64,
```

Comment:
I don't recall but maybe worth it check what we do during the selection of the host for this architecture.

Reply:
We do some fallbacks, I cannot really test it since I don't have hardware, but if it is working on s390x or ppc64le it should work for this architecture as well. The rest of the platform special cases x86, x64 and arm64 (e.g. in blame), but not the other ones. So the change here is pretty much the same as what we did for S390x and Ppc64le.